### PR TITLE
Doc: update WaaG image creation with GVT-d GOP

### PR DESCRIPTION
**Pull request by fuzhongl**
Since GVT-g isn't enable for new platform (i.e TGL EHL CFL),
update the doc about using GVT-d GOP to create WaaG image.

Signed-off-by: fuzhongl <fuzhong.liu@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by dbkinder** [comment]
@fuzhongl I pushed some fixes directly to your branch to remove the doc build errors and some formatting and grammar fixes.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/using_windows_as_uos.rst [hunk 1]
```diff
@@ -72,43 +72,82 @@ Create a raw disk
...
-#. Edit the ``acrn-dm`` command line in ``install_win.sh`` as follows:
-
-   .. note:: Make sure you use GVT-g ``-s 2,pci-gvt -G "$2"`` in the
-      ``acrn-dm`` command line. Currently, we do not support creating a
-      windows image by GVT-d.
-
-   - Change ``-s 3,virtio-blk,./win10-ltsc.img`` to your path to the Windows
-     10 image.
+#. Refer to :ref:`gpu-passthrough` to enable GVT-d GOP feature. Then
+   copy above ``.iso`` files and the build ``OVMF.fd`` to
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:83]:
```suggestion
   copy above ``.iso`` files and the built ``OVMF.fd`` to
```
?

File: doc/tutorials/using_windows_as_uos.rst [hunk 2]
```diff
@@ -72,43 +72,82 @@ Create a raw disk
...
+   #for memsize setting
+   mem_size=4096M
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
+     -s 2,passthru,0/2/0,gpu \
+     -s 8,virtio-net,tap0 \
+     -s 4,virtio-blk,/home/acrn/work/win10-ltsc.img
+     -s 5,ahci,cd:./home/acrn/work/windows.iso \
+     -s 6,ahci,cd:./home/acrn/work/winvirtio.iso \
+     -s 7,passthru,0/14/0,d3hot_reset \
+     --ovmf ./home/acrn/work/OVMF.fd \
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:117]: I do not think we need the dot at the front?
```suggestion
     --ovmf /home/acrn/work/OVMF.fd \
```

File: doc/tutorials/using_windows_as_uos.rst [hunk 3]
```diff
@@ -72,43 +72,82 @@ Create a raw disk
...
-   - Change ``-s 3,virtio-blk,./win10-ltsc.img`` to your path to the Windows
-     10 image.
+#. Refer to :ref:`gpu-passthrough` to enable GVT-d GOP feature. Then
+   copy above ``.iso`` files and the build ``OVMF.fd`` to
+   ``/home/acrn/work``
+#. Prepare WaaG install script
+
+.. code-block:: bash
+
+   vim /home/acrn/work/install_win.sh
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:89]: Probably best to split this in a different code-block, or just mention in the section title that a `/home/acrn/work/install_win.sh` file should be created with the following content.

File: doc/tutorials/using_windows_as_uos.rst [hunk 4]
```diff
@@ -72,43 +72,82 @@ Create a raw disk
...
+   echo "8086 3ea0" > /sys/bus/pci/drivers/pci-stub/new_id
+   echo "0000:00:02.0" > /sys/bus/pci/devices/0000:00:02.0/driver/unbind
+   echo "0000:00:02.0" > /sys/bus/pci/drivers/pci-stub/bind
+   #for memsize setting
+   mem_size=4096M
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
+     -s 2,passthru,0/2/0,gpu \
+     -s 8,virtio-net,tap0 \
+     -s 4,virtio-blk,/home/acrn/work/win10-ltsc.img
+     -s 5,ahci,cd:./home/acrn/work/windows.iso \
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:114]: Do we really need the dot at the beginning of the path? Or should this be
```suggestion
     -s 5,ahci,cd:/home/acrn/work/windows.iso \
```
?

File: doc/tutorials/using_windows_as_uos.rst [hunk 5]
```diff
@@ -72,43 +72,82 @@ Create a raw disk
...
+     -s 6,ahci,cd:./home/acrn/work/winvirtio.iso \
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:115]:
```suggestion
     -s 6,ahci,cd:/home/acrn/work/winvirtio.iso \
```
?

File: doc/tutorials/using_windows_as_uos.rst [hunk 6]
```diff
@@ -72,43 +72,82 @@ Create a raw disk
...
-     ISO.
+#. Run **install_win.sh**
 
-   - Add ``-s 9,ahci,cd:./winvirtio.iso`` to point to your path to the
-     winvirtio iso.
+   .. code-block:: bash
 
-Install Windows 10 by GVT-g
----------------------------
+      cd /home/acrn/work/
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:147]: Do we not need to make the file executable? (i.e. `chmod +x install_win.sh`)

File: doc/tutorials/using_windows_as_uos.rst [hunk 7]
```diff
@@ -172,26 +211,36 @@ Install Windows 10 by GVT-g
...
 
-   .. note::
-      We support GVT-g and GVT-d while launching Windows guest. If you use GVT-g, you can set up Weston in the Service VM, and set up
-      Weston as the desktop environment in the Service VM in order to
-      experience Windows with the AcrnGT local display feature. If you use
-      GVT-d, set **DVMT Pre-Allocated** to **64MB** and set **PM Support**
-      to **Enabled** in the BIOS. Then, only Windows displays.
+      cp /home/acrn/work/install_win.sh  /home/acrn/work/launch_win.sh
+
+   remove following in ``launch_win.sh``
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:228]: Can we be more verbose here?
```suggestion
   Remove the following lines in ``launch_win.sh``
```

**Review by gvancuts — APPROVED** [review]
A few minor things but this looks good to me!

File: doc/tutorials/using_windows_as_uos.rst (hunk 1 above)
Comment (fuzhongl) [reply doc/tutorials/using_windows_as_uos.rst]: Thanks!

File: doc/tutorials/using_windows_as_uos.rst (hunk 3 above)
Comment (fuzhongl) [reply doc/tutorials/using_windows_as_uos.rst]: Done!

File: doc/tutorials/using_windows_as_uos.rst (hunk 4 above)
Comment (fuzhongl) [reply doc/tutorials/using_windows_as_uos.rst]: Forget to remove dot when update it to absolute path.
Thanks!

File: doc/tutorials/using_windows_as_uos.rst (hunk 6 above)
Comment (fuzhongl) [reply doc/tutorials/using_windows_as_uos.rst]: Done!

File: doc/tutorials/using_windows_as_uos.rst (hunk 7 above)
Comment (fuzhongl) [reply doc/tutorials/using_windows_as_uos.rst]: Done!

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by fuzhongl** [comment]
> @fuzhongl I pushed some fixes directly to your branch to remove the doc build errors and some formatting and grammar fixes.

@dbkinder Thanks a lot!
Sorry to forget wget the file before push again.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**